droid-hal-device.inc

```diff
@@ -37,6 +37,7 @@
 %endif
 
 %define __provides_exclude_from ^%{_libexecdir}/droid-hybris/.*$
+%define __provides_exclude_from ^%{_libdir}/droid-devel/.*$
```

Comment:
It's a little unclear why the original exclude exists, but after enabling both excludes both the main and -devel packages break. What about replacing the original (which actually happens now) so the libraries would be provided by the main package and not the -devel which is the case currently. This happens at least on local builds of droid-hal-device.